packages/clients/textLocator/src/plugins/GeometrySearch/store/actions/setupTooltip.ts

```diff
@@ -0,0 +1,50 @@
+import { Tooltip, getTooltip } from '@polar/lib-tooltip'
+import { Feature, Overlay } from 'ol'
+import { vectorLayer } from '../../utils/vectorDisplay'
+import { getPrimaryName } from '../../../../utils/coastalGazetteer/getPrimaryName'
+
+const localeKeys: [string, string][] = [
+  ['h2', 'plugins.geometrySearch.tooltip.title'],
+]
+
+export function setupTooltip({ rootGetters: { map } }) {
+  let element: Tooltip['element'], unregister: Tooltip['unregister']
+  const overlay = new Overlay({
+    positioning: 'bottom-center',
+    offset: [0, -5],
+  })
+  map.addOverlay(overlay)
+  map.on('pointermove', ({ pixel, dragging, originalEvent }) => {
+    if (dragging || ['touch', 'pen'].includes(originalEvent.pointerType)) {
+      return
+    }
+    let hasFeatureAtPixel = false
+    const listEntries: string[] = []
+    map.forEachFeatureAtPixel(
+      pixel,
+      (feature) => {
```

Comment:
A reasonable type is missing

Reply:
A reasonable type was added (first try) https://github.com/Dataport/polar/commit/e602991a93edfeaa80df49716c784904ec433cc1